larq/quantized_variable.py

```diff
@@ -330,6 +366,11 @@ def shape(self):
     def get_shape(self):
         return self.latent_variable.get_shape()
 
+    def __tf_tracing_type__(self, context):
+        if _SUPPORTS_TRACE_TYPE:
+            return QuantizedVariableSpec(self)
+        return NotImplemented
```

Comment:
It's safe to return `NotImplemented` here? What does the caller do with the result?

Reply:
I think so, yes. If the TensorFlow version doesn't support trace type I am pretty sure this function will never get called so it's fine to return `NotImplemented`.